runtime/compilers/rillv1/parse_model.go

```diff
@@ -144,6 +147,31 @@ func (p *Parser) parseModel(node *Node) error {
 		}
 	}
 
+	// hack to mark model as updated when local file changes
+	if inputConnector == "local_file" {
+		path, ok := inputProps["path"].(string)
+		if ok {
+			root, err := p.Repo.Root(ctx)
+			if err != nil {
+				return err
+			}
+			localPaths, err := doublestar.FilepathGlob(filepath.Join(root, path))
+			if err == nil {
+				for _, localPath := range localPaths {
+					localPath = strings.TrimPrefix(localPath, root)
+					resources := p.LocalDataToResourcepath[localPath]
+					if _, ok := p.LocalDataToResourcepath[localPath]; !ok {
+						resources = make(map[string]any)
+						p.LocalDataToResourcepath[localPath] = resources
+					}
```

Comment:
More repetition than necessary:
```suggestion
					resources, ok := p.LocalDataToResourcepath[localPath]
					if !ok {
						resources = make(map[string]any)
						p.LocalDataToResourcepath[localPath] = resources
					}
```